calicoctl/resourcemgr/bgppeer.go

```diff
@@ -15,42 +15,51 @@
 package resourcemgr
 
 import (
-	"github.com/projectcalico/libcalico-go/lib/api"
-	"github.com/projectcalico/libcalico-go/lib/api/unversioned"
-	"github.com/projectcalico/libcalico-go/lib/client"
+	"context"
+
+	api "github.com/projectcalico/libcalico-go/lib/apiv2"
+	client "github.com/projectcalico/libcalico-go/lib/clientv2"
+	"github.com/projectcalico/libcalico-go/lib/options"
 )
 
 func init() {
+	ctx := context.Background()
 	registerResource(
 		api.NewBGPPeer(),
 		api.NewBGPPeerList(),
+		false,
+		[]string{"bgppeer", "bgppeers", "bgpp", "bgpps", "bp", "bps"},
 		[]string{"SCOPE", "PEERIP", "NODE", "ASN"},
 		[]string{"SCOPE", "PEERIP", "NODE", "ASN"},
 		map[string]string{
-			"SCOPE":  "{{.Metadata.Scope}}",
-			"PEERIP": "{{.Metadata.PeerIP}}",
-			"NODE":   "{{.Metadata.Node}}",
+			"SCOPE":  "{{ if eq .Spec.Node `` }}global{{ else }}node{{ end }}",
```

Comment:
Maybe just get rid of the SCOPE field.  For a global-scoped peer you could use the name `(global)` or similar since that isn't a valid name.

Reply:
how about something like this?
```
NAME    PEERIP        NODE          ASN
name1     192.168.1.1   rack1-host1   63400
namester   1.1.1.1       (global)              63444
```